docs/UserGuide.md

```diff
@@ -202,12 +225,12 @@ Examples:
   ![result for 'view 4'](images/viewResult.png)
 * `find Betsy` followed by `viewSchedule 1` views the 1st person in the results of the `find` command.
 
-
-### Viewing persons by tags: `viewGroup`
-Shows the names of friends with the same tag
+### Viewing contacts by tags: `viewGroup`
```

Comment:
```suggestion
### Viewing persons by tags: `viewGroup`
```
This should be persons. Throughout the user guide there isn't really any mentions of contacts